assets/css/style.css

```diff
@@ -289,6 +289,36 @@ html[dir='rtl'] #srcLanguages {
     margin-left: 30px;
 }
 
+/* SpellCheker */
+
+#spellcheckerInput {
+    display: block;
+    height: 131px;
+    overflow: auto;
+    resize: both;
+}
+
+.spellcheckVisible .spellError {
+    border-bottom: 1px solid #f00;
+}
+
+.popover-content { /* sass-lint:disable-line class-name-format */
```

Comment:
Can we make this selector more specific?

Reply:
`.popover-content` doesn't seem to be used anywhere else in the code.